ledger/src/blockstore.rs

```diff
@@ -1408,6 +1419,17 @@ impl Blockstore {
         Ok(insert_results.completed_data_set_infos)
     }
 
+    // Test-only function.
+    pub fn insert_shreds(
```

Comment:
please add appropriate cfg guard (test_context_only_utils or at least name it xxx_for_tests

Reply:
Despite how the diff looks like, this is not a new function.
`cfg(test)` does not work here because the function is called outside of the crate.
Also there are many many instances of this function, and if we want to rename it that should happen in a separate PR to keep this PR small.

Reply:
I'll agree that we don't want to muddy the water with a large rename; could use `dev-context-only-utils` to get around the can't-call-test-function-in-other-crate issue